homeassistant/components/airnow/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "AirNow",
+  "config": {
+    "step": {
+      "user": {
+        "title": "AirNow",
+        "description": "Set up AirNow air quality integration. To generate API key go to https://docs.airnowapi.org/account/request/",
+        "data": {
+          "name": "Name of the Entity",
```

Comment:
We usually use `Name of the integration`.

Reply:
Updated in 0fdbbff